### DISCUSSION TITLE
Когда заработает снова?

### DISCUSSION THREAD

**Discussion by Olezha180** [anticensority/runet-censorship-bypass#310] [Q&A]
Когда заработает снова?

**Comment by ilyaigpetrov** [comment]
Ответная мера на ежедневную блокировку РКН зависит от разработчика проекта "АнтиЗапрет".
Он немного занят, но я ему напоминаю время от времени об этой задаче.
Подпишитесь на новости: https://github.com/anticensority/runet-censorship-bypass/issues/10.

**Reply by Olezha180** [reply]
> Ответная мера на ежедневную блокировку РКН зависит от разработчика проекта "АнтиЗапрет". Он немного занят, но я ему напоминаю время от времени об этой задаче. Подпишитесь на новости: #10.

У меня на телефоне стоит "Super VPN" (зелёная иконка у него), который вообще никак не банится РКН'ом. Почему ваше приложение тогда банится?
